modules/react-native-desktop-config/index.js

```diff
@@ -4,15 +4,15 @@ const NativeModules = require('react-native').NativeModules;
 
 class DesktopConfig {
 
-  static getLoggingEnabled(callbackFn) {
-    NativeModules.DesktopConfigManager.getLoggingEnabled(
-      (enabled) => {
-        callbackFn(enabled);
+  static getValue(name, callbackFn) {
+    NativeModules.DesktopConfigManager.getValue(name,
```

Comment:
Can it be simplified to this?

```suggestion
    NativeModules.DesktopConfigManager.getValue(name, callbackFn);
```

Reply:
Funny, I thought that if we keep using React Native callbacks for native-to-js communication, then the answer would be "no". Per [RN docs](
https://facebook.github.io/react-native/docs/native-modules-ios#callbacks), `A native module should invoke its callback exactly once.` And that by wrapping in an extra lambda fn, we ensure that callback fns are not getting reused.
But I was wrong, this does not apply in this case (it affected e.g. https://github.com/status-im/status-react/pull/7038 so it had to be changed to use events) Thanks for noticing this @PombeirP !